spec/Observable-spec.ts

```diff
@@ -59,6 +59,20 @@ describe('Observable', () => {
     }).to.throw();
   });
 
+  it('should rethrow if sink has syncErrorThrowable = false', () => {
+    const observable = new Observable(observer => {
+      observer.next(1);
+    });
+
+    const sink = Subscriber.create(() => {
+      throw 'error!';
+    });
+
+    expect(() => {
+      observable.subscribe(sink);
+    }).to.throw('error!');
+  });
+
```

Comment:
this test fails without the two other changes, so I made it as a separate commit so it's a bit easier for someone to confirm